auth_register creation

thomrey023 opened the pull request:
Created register_screen.dart and its corresponding files and functions, naa pako mga questions sa other functionalities pero ig balik nlng nato sa office ko mg ask nimo.

thomrey023 committed:
auth_register creation

Created register_screen.dart and its corresponding files and functions

chiekkored commented:
Thom! Align Center ang text!

<img width="428" alt="image" src="https://user-images.githubusercontent.com/70371749/193403390-26eca517-c2cc-4598-adae-aff2cea4a311.png">

Nya I think need ug loading nga UI inigclick sa Register button. Kay basin makapila ug click sa user ang button ba kay walay nahitabo progress after click sa register button

@thomrey023

thomrey023 commented:
Ay oh!! Nalimot ko ug center sa text ana. Gegegege, himu.an nko ug loading na UI, apilon nlng pud nko sa login

thomrey023 commented:
![image](https://user-images.githubusercontent.com/60767992/193411916-32db82e3-9a76-4f3b-ab84-d883d3bea765.png)
Nka Alignment.center nani sya chiekk, same2 ra sya ug result. Nya ga himo ko ug new customheadertext nga way ellipsis

thomrey023 committed:
Register and Login

Added Login and Register functions and additional functions, also included logout
